.github/workflows/bigquery.yml

```diff
@@ -0,0 +1,27 @@
+name: BigQuery
```

Comment:
are you setting things up mostly as *separate* workflows? is there a reason to do / not to do this?

Reply:
I think bigquery tests require a key as a github secret, which is only available when merging to master, not in PRs. When a job runs is defined at file level (not at job level), that's why I have it separate. I think the alternative could be to skip every step individually and have everything together, but for now this felt more neat.

Reply:
kk yep